src/frontend/js/widgets/SyllabusCourseRunsList/components/CourseRunEnrollment/index.tsx

```diff
@@ -291,30 +300,37 @@ const CourseRunEnrollment: React.FC<CourseRunEnrollmentProps> = (props) => {
         </React.Fragment>
       );
     case step === Step.ENROLLED:
-      return courseRun.starts_in_message ? (
+      return isStarted ? (
+        <div>
+          <Button
+            href={courseRun.resource_link}
+            className="course-run-enrollment__cta"
+            fullWidth={true}
+          >
+            <FormattedMessage {...messages.goToCourse} />
+          </Button>
+          <div className="course-run-enrollment__helptext">
+            <FormattedMessage {...messages.enrolled} />
+            {(canUnenroll || true) && (
```

Comment:
No quit sure to understand why you put `|| true` ?

Reply:
Debug residue ^^'